deployments/core.go

```diff
@@ -204,13 +204,16 @@ func (core *Core) createCoreDeployment(giteaURL string) error {
 	re := regexp.MustCompile(`__GITEA_URL__`)
 	renderedFileContents := re.ReplaceAllString(string(fileContents), giteaURL)
 
+	re = regexp.MustCompile(`__TEKTON_DASHBOARD_URL__`)
+	renderedFileContents = re.ReplaceAllString(string(renderedFileContents), tektonURL)
```

Comment:
I am ok with that, but I would consider creating a config map with those values and making the deployment read it from there.